### PR TITLE
Fix fine-tuning callback test

### PR DESCRIPTION
**Pull request by awaelchli**
## What does this PR do?

Fixes a test for the FineTuningCallback

The fine-tuning callback test was not correct, because
1. I didn't use the backbone at all
2. batchnorm works only with batch size >= 2

@tchaton 

Encountered in #5507

## Before submitting
- [ ] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [ ] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [ ] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [ ] Did you make sure to update the documentation with your changes? (if necessary)
- [ ] Did you write any new necessary tests? (not for typos and docs)
- [ ] Did you verify new and existing tests pass locally with your changes?
- [ ] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [ ] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified
 - [x] **Check that target branch and milestone match!**

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Review by Borda — COMMENTED** [review]
and add assets to check that the backbone is used and born... as you wrote in the description, just in case someone touches it in the future...

**Comment by awaelchli** [comment]
good suggestion but I would appreciate if @tchaton did this as he is the author of the callback

**Review by tchaton — APPROVED** [review]
LGTM !

**Review by Borda — APPROVED** [review]

**Review by SkafteNicki — APPROVED** [review]

**Comment by Borda** [comment]
@tchaton mind rebase?

**Comment by Borda** [comment]
@awaelchli seems after rebasing all other changes were already on feat 1.2

**Review by carmocca — APPROVED** [review]
